README.md

```diff
@@ -218,6 +219,11 @@ Contributions of any kind welcome!
 [**Read The Docs**](https://testing-library.com/angular) |
 [Edit the docs](https://github.com/testing-library/testing-library-docs)
 
+## FAQ
+
+##### I am using Reactive Forms and the `jest-dom` matcher `toHaveFormValues` always returns an empty object or there are missing fields. Why?
```

Comment:
How about mentioning the directive proposed by @timdeschryver?

Reply:
I'm going to merge this one already, because it helps other devs.
Feel free to update this and mention the directive @rafaelss95 .